### PR TITLE
Update documentation on geodesics

### PR DESCRIPTION
**Pull request by cffk**
Replacing an ancient page on geodesic calculations with something more helpful to the beginning user.

### CODE REVIEW

File: docs/source/geodesic.rst [hunk 1]
```diff
@@ -1,219 +1,217 @@
...
+
+Consider a ellipsoid of revolution with equatorial radius *a*, polar
+semi-axis *b*, and flattening *f* = (*a* − *b*)/*a* .  Points on
+the surface of the ellipsoid are characterized by their latitude φ
+and longitude λ.  (Note that latitude here means the
+*geographical latitude*, the angle between the normal to the ellipsoid
+and the equatorial plane).
+
+The shortest path between two points on the ellipsoid at
+(φ\ :sub:`1`, λ\ :sub:`1`) and (φ\ :sub:`2`, λ\ :sub:`2`)
```
Comment (kbevers) [inline docs/source/geodesic.rst]: I think that
 
```
:math: `lambda`

```

gives a better results than simply using λ.  This applies many places obviously.

Comment (hobu) [reply docs/source/geodesic.rst]: yes, especially when the Sphinx->latex->PDF conversion happens.

File: docs/source/geodesic.rst [hunk 2]
```diff
@@ -1,219 +1,217 @@
...
+is called the geodesic.  Its length is
+*s*\ :sub:`12` and the geodesic from point 1 to point 2 has forward
+azimuths α\ :sub:`1` and α\ :sub:`2` at the two end
+points.  In this figure, we have λ\ :sub:`12` =
+λ\ :sub:`2` − λ\ :sub:`1`.
+
+    .. raw:: html
+
+       <center>
+         <img src="https://upload.wikimedia.org/wikipedia/commons/c/cb/Geodesic_problem_on_an_ellipsoid.svg"
```
Comment (kbevers) [inline docs/source/geodesic.rst]: @hobu has previously warned against using svg's. I believe they render fine in the HTML versions, but success is not guaranteed in the PDF.

File: docs/source/geodesic.rst [hunk 3]
```diff
@@ -1,219 +1,217 @@
...
+* The solution of the inverse problem is always found.  (Vincenty's
+  method fails to converge for nearly antipodal points.)
+* The routines calculate differential and integral properties of a
+  geodesic.  This allows, for example, the area of a geodesic polygon to
+  be computed.
+
+References
+----------
+
+* F. W. Bessel,
```
Comment (kbevers) [inline docs/source/geodesic.rst]: It'd be nice to have these in `docs/references.rst` as well. To keep some sort of consistency across the documentation.

File: docs/source/geodesic.rst (hunk 2 above)
Comment (hobu) [reply docs/source/geodesic.rst]: We don't want to use SVG in the docs unless we also have a PNG, or we have a pipeline to go SVG->PNG automatically as part of the Sphinx processing. Sphinx HTML output is fine with SVG, but PDF is challenging.

File: docs/source/geodesic.rst [hunk 4]
```diff
@@ -1,219 +1,217 @@
...
+         <img src="https://upload.wikimedia.org/wikipedia/commons/c/cb/Geodesic_problem_on_an_ellipsoid.svg"
+              alt="Figure from wikipedia"
+              width="250">
+       </center>
+
+A geodesic can be extended indefinitely by requiring that any
+sufficiently small segment is a shortest path; geodesics are also the
+straightest curves on the surface.
+
+Solution of geodesic programs
```
Comment (busstoptaktik) [inline docs/source/geodesic.rst]: programs -> problems
(I suppose)

Comment (cffk) [reply docs/source/geodesic.rst]: Yup.  I will change.

File: docs/source/geodesic.rst (hunk 3 above)
Comment (cffk) [reply docs/source/geodesic.rst]: OK, I'll move them over.

File: docs/source/geodesic.rst (hunk 2 above)
Comment (cffk) [reply docs/source/geodesic.rst]: I'll just substitute the corresponding PNG file from wikimedia.  Is that OK?

File: docs/source/geodesic.rst (hunk 1 above)
Comment (cffk) [reply docs/source/geodesic.rst]: All right, I'll use this style for all math.

**Review by busstoptaktik — APPROVED** [review]

**Comment by kbevers** [comment]
Thanks, Charles.

Also merged into the `docs-release-4.10.0`-branch: https://kbevers.github.io/geodesic.html

**Pull request merged**